src/index.js

```diff
@@ -10,17 +10,19 @@ const deviceStorage = {
 	/**
 	 * Get a one or more value for a key or array of keys from AsyncStorage
 	 * @param {String|Array} key A key or array of keys
+	 * @param {String|Array} [defaultValue] Returned if the key(s) are null
 	 * @return {Promise}
 	 */
-	get(key) {
+	get(key, defaultValue) {
 		if(!Array.isArray(key)) {
 			return AsyncStorage.getItem(key).then(value => {
-				return JSON.parse(value);
+				return JSON.parse(value) || defaultValue;
```

Comment:
Let's change this to check for specifically a null value being returned by `JSON.parse(value)`. We probably wouldn't want to return the `defaultValue` if the parsed value is anything falsy (`0`, `false`, etc).